dev-util/cepl/cepl-8.3.0.ebuild

```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+DESCRIPTION="An interactive C and C++ read–eval–print loop."
+HOMEPAGE="https://github.com/alyptik/cepl"
+SRC_URI="https://github.com/alyptik/cepl/releases/download/v${PV}/cepl-${PV}.tar.gz"
+S="${WORKDIR}"
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+RDEPEND="
+	sys-libs/readline
+	dev-libs/libelf
+	sys-libs/ncurses
+	"
+DEPEND="${RDEPEND}"
+
```

Comment:
```sh
src_compile() {
	tc-export CC
	export {C,LD}FLAGS
	emake cepl
}
```
Inherit the `toolchain-funcs` eclass and add something like this to ensure that these compiler vars are actually exported for `make` to pick them up (the PMS doesn't guarantee that), then set the explicit target to avoid calling `make check` during the compile phase instead of the test phase.